### ISSUE TITLE
Site takes a long time to load

### ISSUE THREAD

**Issue by itselijahciali** [nicksawhney/me#1]
**This site's file structure is making the site essentially impossible to use.** At first I thought it was an issue with the preloader, but apparently it wasn't. Take a look at my DevTools:

![image](https://user-images.githubusercontent.com/16740188/105571915-55cd1000-5d19-11eb-9ad4-b9e6ab8aa006.png)

As you can see, it takes over **3.5 minutes** to load all of the content in this site. It must be some kind of huge file, right? Turns out the site is full of unfortunate optimizations, such as [this file](https://nicksawhney.github.io/fare_evasion/figs/qn031.png). Might seem simple enough, loads in a second or two. The problem is that there are hundreds of these pictures, most of which are associated with the "Fare Evasion" project. Downloading hundreds of these high resolution pictures in parallel from an already slow GitHub server through jQuery doesn't just cause problems for you, it also can rack up data bills for people viewing on mobile or on a metered broadband connection. This leads to over 1 minute download times for each file!

**My recommendation:** dynamically load "Fare Evasion" images and try to make them rendered locally with SVGs. If you can't, try heavy compression and **do not** have them download individually.

**Comment by nicksawhney** [comment]
You're right, and I've been meaning to get to this for a while. The images were created to be accessible for research papers more so than the web, though I'm working on a version which uses dynamically generated plotly charts instead of images of pie charts. Should lower the load time immensely! 
Thanks for the issue submission.

**Comment by nicksawhney** [comment]
Also the main issue here isn't really the site itself, but rather an HTML iframe loading the github pages version of the `fare-evasion` repo on my profile. If you know of a way to make that site wait to load until the user scrolls to it, I'd be really interested in pushing it right away

**Comment by itselijahciali** [comment]
Can you make the iframe load once the site is ready so that people can view the site, maybe with a loading animation over the iframe?
